README.md

```diff
@@ -45,20 +52,25 @@ Depending on the available number of CPU cores and how parallelizable the depend
 specification, one may be able to specify more concurrent packages (as compared to `2` in the above example).
 
 
-## Building THAPI from Source
+## Building THAPI manually
 
 You can use Spack to install the required dependencies and set up the environment to build THAPI from source:
 ```bash
 spack build-env thapi bash
 ```
 This will spawn a new shell with the correct environment variables set for building THAPI manually.
 
+## Known issues
+
+THAPI spack package (and some of its dependencies) is known to run into build failures with NVHPC SDK
+(available on ALCF Polaris for example). Please make sure to use GNU programming environment whenever
+possible.
 
-## Miscelanous
+## Miscellaneous
 
-### How to Install Spack
+### How to install Spack
```

Comment:
if you change this one, you need to change line  6 : ) 

but in the US they capitalize all the work in title (https://www.grammarly.com/blog/punctuation-capitalization/capitalization-in-titles/) but the rule at complex :p . Just let try to be coherent for my poor OCD. Sorry!